go.mod

```diff
@@ -3,21 +3,26 @@ module github.com/e-money/em-ledger
 go 1.12
 
 require (
-	github.com/cosmos/cosmos-sdk v0.39.1
-	github.com/gorilla/mux v1.7.4
-	github.com/onsi/ginkgo v1.7.0
-	github.com/onsi/gomega v1.4.3
-	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v1.0.0
-	github.com/spf13/viper v1.6.3
-	github.com/stretchr/testify v1.6.1
-	github.com/tendermint/go-amino v0.15.1
-	github.com/tendermint/tendermint v0.33.7
-	github.com/tendermint/tm-db v0.5.1
+	github.com/cosmos/cosmos-sdk v0.41.3
+	github.com/gogo/protobuf v1.3.3
+	github.com/golang/protobuf v1.4.3
+	github.com/gorilla/mux v1.8.0
+	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/onsi/ginkgo v1.14.0
+	github.com/onsi/gomega v1.10.1
+	github.com/rakyll/statik v0.1.7
+	github.com/regen-network/cosmos-proto v0.3.1
+	github.com/spf13/cast v1.3.1
+	github.com/spf13/cobra v1.1.1
+	github.com/spf13/pflag v1.0.5
+	github.com/spf13/viper v1.7.1
+	github.com/stretchr/testify v1.7.0
+	github.com/tendermint/tendermint v0.34.8
+	github.com/tendermint/tm-db v0.6.4
 	github.com/tidwall/gjson v1.3.2
 	github.com/tidwall/sjson v1.0.4
+	google.golang.org/genproto v0.0.0-20210114201628-6edceaf6022f
+	google.golang.org/grpc v1.35.0
 )
 
-// replace github.com/cosmos/cosmos-sdk => ./tmpvendor/cosmos-sdk
-
-// replace github.com/tendermint/tendermint => ./tmpvendor/tendermint
+replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
```

Comment:
// Add replace for stable grpc dep as per sdk and documented 
https://github.com/grpc/grpc-go/issues/4192
https://docs.cosmos.network/v0.41/core/grpc_rest.html#grpc-server
replace google.golang.org/grpc => google.golang.org/grpc v1.33.2

Reply:
👍  I did not know this constraint